ar_exercise/exercise.rb

```diff
@@ -51,6 +114,22 @@ class Apartment < ActiveRecord::Base
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+
+puts "create 3 new apartments"
+Apartment.create do |apt|
+	puts "enter apt address"
+	apt.address = gets.chomp
+	puts "enter monthly rent"
+	apt.monthly_rent = gets.chomp.to_i
+	puts "enter square feet "
+	apt.sqft = gets.chomp.to_i
+	puts "enter num beds "
+	apt.num_beds = gets.chomp.to_i
+	puts "enter num bathrooms"
+	apt.num_baths = gets.chomp.to_i
+end
```

Comment:
Instead of asking for user input, a much better way to do this would be to just hardcode them in.
```rb
# using a local variable so I can do stuff with them later
apartment1 = Apartment.create(address: "1600 Pennsylvania Ave", "more key value pairs etc. etc." )
```